tests/main/interfaces-netlink-audit/task.yaml

```diff
@@ -5,6 +5,10 @@ details: |
 
     The test-snapd-netlink-audit snap creates a netlink socket and binds it. 
 
+# Ubuntu 14.04-64 is skipped due to the capability CAP_AUDIT_READ
+# is not available in its kernel
+systems: [-ubuntu-14.04-*]
```

Comment:
I thought that Ubuntu 14.04 systems needed the lts kernel, so therefore 4.4. Is it not using the right kernel?

Reply:
Maybe one of the layers is out of date (libcap?). I agree with @jdstrand here, it should not be any different.